### PR TITLE
obs space monitoring

### PR DESCRIPTION
**Pull request by guillaumevernieres**
## Description
This PR adds a tool to plot observation space information available in a ioda file. 
It will produce things like this:

```console
./godas_plotobs.py -f ~/Volumes/orion/runs/s2s/final-0.25/obs_out/2020/2020121218/ctrl/adt_*.nc -g ombg -v absolute_dynamic_topography -b -.2 .2 -c jet -q 0
```

adt-omb-example

```console
./godas_plotobs.py -f ~/Volumes/orion/runs/s2s/final-0.25/obs_out/2020/2020121218/ctrl/sst_viirs_npp_l3u_so025.20201212.nc -g ombg -v sea_surface_temperature -b -1 1 -c jet -q 0
```
sst-omb-example

### CODE REVIEW

**Review by ShastriPaturi — APPROVED** [review]
Looks good. Other options can be added.

**Comment by mjhossen** [comment]
works perfectly. add some features in the plt.savefig such as  bbox_inches='tight', pad_inches = 0.02

**Comment by guillaumevernieres** [comment]
> works perfectly. add some features in the plt.savefig such as bbox_inches='tight', pad_inches = 0.02

Let's not push into each other's branch without doing a PR or making sure with the author of the branch that it is ok @mjhossen. It's not critical in this case, but it certainly is a bad habit.

**Comment by mjhossen** [comment]
I agree. I put my scripts in the new branch: feature/spaceplot_tool.

File: monitoring/godas_plotobs.py [hunk 1]
```diff
@@ -0,0 +1,213 @@
+#!/home/gvernier/anaconda3/bin/python
```
Comment (ShastriPaturi) [inline monitoring/godas_plotobs.py]: comment the line

**Review by ShastriPaturi — CHANGES_REQUESTED** [review]

File: monitoring/godas_plotobs.py (hunk 1 above)
Comment (guillaumevernieres) [reply monitoring/godas_plotobs.py]: @ShastriPaturi , done.

**Review by ShastriPaturi — APPROVED** [review]
Thanks

**Comment by mjhossen** [comment]
good to go.

**Pull request merged**